microsite/docusaurus.config.js

```diff
@@ -68,6 +68,12 @@
       },
     ],
   ],
+  markdown: {
+    preprocessor({ filePath, fileContent }) {
+      // Replace all HTML comments with emtpy strings as these are not supported by MDXv2.
+      return fileContent.replace(/<!--.*?-->/gs, '');
```

Comment:
Nice, is this perhaps a replacement for `scripts/pre-build.js` ?

Reply:
Yep - now html docs are not supported anywhere, so I just strip them everywhere rather than just the API docs. But we can of course still write comments where needed, as we do have some examples where we need them.

Reply:
Love it